### DISCUSSION TITLE
Problems with lineamouse mouse and trackpad in mac book

### DISCUSSION THREAD

**Discussion by seahousemomo** [linearmouse/linearmouse#526] [Q&A]
When linarmouse is turned on, the speed of the mouse is adjustable and very cool to use. But the speed of the touchpad can not be adjusted through the system settings, no wheel how to adjust the tracking speed of the touchpad, but the use of the touchpad slide can only be in a fixed speed, the problem affects my use, help the gods to help solve.

My mac information is as shown
<img width="487" alt="image" src="https://github.com/linearmouse/linearmouse/assets/138654810/e553b91f-7669-4fc1-9d54-adf3ff0e8715">

**Comment by seahousemomo** [comment]
The version of lineamouse is 0.9.2

**Comment by lujjjh** [comment]
Thanks for your feedback.

In LinearMouse, you can modify the tracking speed of your trackpad by adjusting the pointer acceleration. Essentially, they are the same thing.

On M2 MacBooks, it's possible that it will show up as "(unknown)". For more information, please refer to issue #270.

---

在 LinearMouse 中，你可以通过调整加速度来修改触控板的跟踪速度，它们是同一个东西。

M2 MacBook 里可能会显示为“(unknown)”，这个问题可以参考 #270.

**Reply by seahousemomo** [reply]
Thank you for your answer, based on your answer I see the touchpad and mouse option in Linearmouse. Thank you very much.

But you can adjust the touchpad and mouse options to two pages, it will give the user a more intuitive feeling of operation, a little bit of advice.
<img width="850" alt="image" src="https://github.com/linearmouse/linearmouse/assets/138654810/d5bff7d9-909c-4121-9413-5e2a41eb0760">
